Makefile

```diff
@@ -342,7 +345,6 @@ help:
 	echo "  format                     Re-format source code to conform style settings enforced by"
 	echo "                             isort and black tools."
 	echo "  prepare-changelog          Add an autobuild version entity to changelog"
-	echo "  prepare-version            Update version based on latest commit"
```

Comment:
Replaced with `generate-version` and it's internal target. It's better to not add it to the help.